### ISSUE TITLE
Connection does not work after dss restart or after some time

### ISSUE THREAD

**Issue by KarlKiel** [ioBroker/ioBroker.digitalstrom#17]
I am not exactly sure if it happens after a certain time or only when the DSS is restarted.
I figured out several times now that suddenly the connection to the DSS server does not work correctly anymore - values do not get refreshed in iobroker and commands done in iobroker will not be reflected in DS. Only a restart of the adapter brings a working connection back. The last values nevertheless stay inside the system and iobroker tells that the connector was connected successfully (green connection icon) - but actually there is no successful exchange possible any longer.
Would be great if the adapter was able to check the connectivity regularly (maybe via a regularly updated value that every DS system delivers - like e.g. sunazimuth) and reestablish connection correctly if values do not change any more... - as a full restart will take a lot of time...
Version is 2.0.5
BR, Arne

**Comment by Apollon77** [comment]
Then please enable debug log ( sorry but this will be MUCH LOG!!) because I need debug log from when it stops working whats the reason ...

**Comment by KarlKiel** [comment]
Hi @Apollon77,
At least now it is clear that successful communication really breaks at restart of the DSS.
Attached is the log showing the following:
15:08 - the iobroker DS adapter instance was started
15:31 - the iobroker DS adapter was up and running, connected to DSS and working correctly - I have letting it run now for 15 minutes to show successful operation
15:44 - I restarted the DSS - After the restart (successfully restarted at ~15:47 I gave another 15 minutes for you to see what is happening after the restart - e.g. values are not being updated now anymore.

I hope that this is going to help to find the cause for this behavior - THANKS in advance for your support!

BR,

Arne

[dssRestart.txt](https://github.com/ioBroker/ioBroker.digitalstrom/files/4831813/dssRestart.txt)
